devel/cdecl/Portfile

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+name                cdecl
+version             3.1.3
+categories          devel
+platforms           darwin
+maintainers         joelburton.com:joel
```

Comment:
Please add you github handle
```
maintainers         {joelburton.com:joel @joelburton}
```
and potentially a keyword `openmaintainer` if you are ok with other developers doing minor changes to the port.